lib/classes/Utils.js

```diff
@@ -100,6 +100,17 @@ class Utils {
     return filePaths;
   }
 
+  copyDirContentsSync(srcDir, destDir) {
```

Comment:
Opps good catch! coveralls didnt help there since this code is actually covered by other tests